CMakeLists.txt

```diff
@@ -890,9 +890,11 @@ ENDIF()
 SUBDIRS(src)
 
 IF(NOT QMC_BUILD_SANDBOX_ONLY)
-  SUBDIRS(tests)
-  SUBDIRS(examples)
-  IF(IS_DIRECTORY "${PROJECT_SOURCE_DIR}/nexus")
+  IF( NOT QMC_DISABLE_SLOW_TESTS )
```

Comment:
Here and the above redefining functions as print are not disabling tests in a consistent way but more "ad hoc".
I'm OK with your current choice of both places and we may learn what minimal testing to keep over time.
I personally would like to keep all the deterministic integration tests in the minimal set.

Reply:
It's a workaround, the commands are "broken" from a developer productivity standpoint. But its a workaround that anyone improving the build (outside of those commands), debugging, or refactoring will save a ton of time with. I'll update the description of the option to explain it is for development.

A better workaround could take TEST labels into account.